thunder/tests/test_dynamo.py

```diff
@@ -788,6 +788,9 @@ def find_target_module(model, target_module_name):
     decorators=(pytest.mark.parametrize("use_pytest_benchmark", (True, False), ids=("benchmark", "repro")),),
 )
 def test_dynamo_reproducer_2graph(executor, device: str, dtype: dtypes.dtype, use_pytest_benchmark, tmp_path):
+    if IS_WINDOWS and use_pytest_benchmark:
+        pytest.skip("torch.compile: Compiler: cl is not found.")
```

Comment:
For this skip comment, let's link to the issue (or file an issue and then link to it if there isn't one) for torch.compile tests being disabled on Windows and say "Skipping on Windows because this uses torch.compile (see <link to issue>)"